javagen/src/main/java/com/azure/autorest/util/returntype/ReturnTypeDescriptionAssembler.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+
+package com.azure.autorest.util.returntype;
+
+import com.azure.autorest.model.clientmodel.GenericType;
+import com.azure.autorest.model.clientmodel.IType;
+import com.azure.autorest.model.clientmodel.PrimitiveType;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReturnTypeDescriptionAssembler {
+    private static final ClientLogger LOGGER = new ClientLogger(ReturnTypeDescriptionAssembler.class);
```

Comment:
Use the `PluginLogger` instead.

autorest.java integrates with autorest, and had to use its logging system.

Reply:
Switched to `PluginLogger`. Thanks for pointing out!